specification/components/r4/schemas/responses/searchForService/200Response.yaml

```diff
@@ -25,15 +27,12 @@ content:
         meta:
           type: object
           required:
-            - profile
+            - lastUpdated
           properties:
-            profile:
-              type: array
-              items:
-                type: string
-                enum:
-                  - 'https://fhir.hl7.org.uk/StructureDefinition/UKCore-Bundle'
-                example: 'https://fhir.hl7.org.uk/StructureDefinition/UKCore-Bundle'
+            lastUpdated:
+              type: string
+              format: date-time
+              example: '2021-08-04T14:35:00.000Z'
```

Comment:
I assume this is OK, because the bundle coming from the back-end always has a lastUpdated? So even though it wasn't documented, we were already sending it back in the response?

Reply:
that's correct, the HealthcareServiceHelper#searchServiceByIds populates the lastUpdated from the bundle